src/sentry/web/frontend/auth_login.py

```diff
@@ -69,6 +69,7 @@ def run_callbacks(self, request: Request) -> dict:
 additional_context = AdditionalContext()
 
 
+# TODO(hybridcloud) Make this view control silo only.
```

Comment:
The issue is that we never added a silo limit decorator to django views, which has contributed to some issues.

Reply:
I think we might need decorators on django views to prevent curious customers from using the application in ways that won't work.